analysis/filter/StructureModifier/src/test/kotlin/de/maibornwolff/codecharta/filter/structuremodifier/SubProjectExtractorTest.kt

```diff
@@ -80,4 +82,15 @@ class SubProjectExtractorTest {
         val edges = result.edges
         Assertions.assertThat(edges).isEmpty()
     }
+
+    @Test
+    fun `Subproject still contains all necessary attributeDescriptors`() {
```

Comment:
Small remark: Keep the given/when/then pattern in mind. I would always include the WHEN and the THEN in the method name.This makes it clear to every other developer whats going on here.

Reply:
I will change this name.